WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/LicensesViewController.swift

```diff
@@ -34,7 +34,7 @@ private extension LicensesViewController {
     /// Set the title and back button.
     ///
     func configureNavigation() {
-        title = NSLocalizedString("Licenses", comment: "Licenses (information page title)")
+        title = NSLocalizedString("Software Licenses", comment: "Software Licenses (information page title)")
```

Comment:
Should be changed to "Third Party Licenses" to align with future plans for Unified About Screen pc5zdG-rB-p2

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5463/commits/b91fc6e3d0064e4073c5559fe760b998aae4f638